pkg/apis/istio/v1alpha1/values_types.proto

```diff
@@ -295,6 +316,12 @@ message GalleyConfig {
 
   // Enable analysis and status update in Galley
   google.protobuf.BoolValue enableAnalysis = 8;
+
+  // K8s rolling update strategy
+  TypeIntOrStringForPB rollingMaxSurge = 9;
```

Comment:
Why is this in Galley config and not the general K8s?

Reply:
Because values.yaml was poorly designed :-)